### PR TITLE
Linear Search Algorithm in Python

### PR DESCRIPTION
**Pull request by jayantaadhikary**
As assigned in the issues section, I completed the Linear Search Algorithm problem using Python.

### CODE REVIEW

File: Linear Search Algorithm/linear_search.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def linearSearch(array, n, x):
+    for i in range(0, n):
+        if (array[i] == x):
+            return i
+    else:
+        return -1
+
+myArray = []
```
Comment (rishabh-live) [inline Linear Search Algorithm/linear_search.py]: the variable name should be more specific. Don't name variables as myArray, Array, etc

**Review by rishabh-live — CHANGES_REQUESTED** [review]
Also, Add Some Documentation to your code so that viewers are able to understand it

**Comment by rishabh-live** [comment]
@jayantaadhikary don't close the request I will alot you tag after merging, the PR may not be aloted the accepted tag and ultimately your contribution won't be counted there.

**Comment by jayantaadhikary** [comment]
> @jayantaadhikary don't close the request I will alot you tag after merging, the PR may not be aloted the accepted tag and ultimately your contribution won't be counted there.

Okay. I am adding the comments and changing the variable names

**Comment by jayantaadhikary** [comment]
Could you check the latest commit? Is it okay now?

**Review by rishabh-live — APPROVED** [review]
- [x] Approved

**Pull request merged**